### ISSUE TITLE
Cannot run test in linux

### ISSUE THREAD

**Issue by wirekang** [RobinBlomberg/kysely-codegen#51]
OS: ubuntu 22.40

Steps:
1. clone repository
2. pnpm install
3. pnpm run test

This line https://github.com/RobinBlomberg/kysely-codegen/blob/ce4e50bbbcb1c35000aff6b09c0e4a2d6700ed90/src/tests/e2e.test.ts#L27

causes error. If I changed it to any relative file Name such as `sqliteDB`, it works well.

**Comment by RobinBlomberg** [comment]
Thanks! I haven't made sure all the tests can be run on any machine, but it's probably time to ensure that they can be (using Docker).

**Comment by wirekang** [comment]
https://github.com/WiseLibs/better-sqlite3/blob/master/docs/api.md#new-databasepath-options

> You can create a temporary database by passing an empty string (or by omitting all arguments).

How about passing empty string?

**Comment by RobinBlomberg** [comment]
I still think Docker may be a good idea so users don't have to install SQLite3 externally, but `''` (temporary database) or `:memory:` (in-memory database) should be the next best thing.

**Comment by wirekang** [comment]
I'm confusing. I've never installed sqlite3. I think `better-sqlite3` package includes sqlite3 runtime itself.

```
╚═ sqlite3
Command 'sqlite3' not found, but can be installed with:
sudo apt install sqlite3


╚═ sqlite
Command 'sqlite' not found, but can be installed with:
sudo apt install sqlite

╚═ sudo npm list -g
/usr/lib
├── corepack@0.15.1
├── npm@8.19.3
├── pnpm@7.20.0
└── yarn@1.22.19
```

**Comment by RobinBlomberg** [comment]
Oh good point... That's probably true. Maybe `:memory:` is sufficient then. 👀

**Comment by wirekang** [comment]
empty connectionString or `:memeory:` will causes the error:
```
[ERROR] 18:42:06 AssertionError: Expected values to be strictly equal:
+ actual - expected

+ 'export interface DB {}\n'
- 'export interface UserTest {\n' +
-   '  id: string | null;\n' +
-   '  userStatus: string | null;\n' +
-   '}\n' +
-   '\n' +
-   'export interface DB {\n' +
-   '  userTest: UserTest;\n' +
-   '}\n'
```

It may have something to do with data persistence.

**Comment by RobinBlomberg** [comment]
That is indeed the case with the current tests, but I think the tests (all of them) need to be written to just be temporarily persistent. So some more work needs to be made.

**Comment by RobinBlomberg** [comment]
Fixed in `kysely-codegen@0.9.0`!
